internal/kafka/command_link_describe.go

```diff
@@ -10,14 +10,20 @@ import (
 )
 
 type linkOut struct {
-	Name               string `human:"Name" serialized:"link_name"`
-	TopicName          string `human:"Topic Name" serialized:"topic_name"`
-	SourceCluster      string `human:"Source Cluster" serialized:"source_cluster,omitempty"`
-	DestinationCluster string `human:"Destination Cluster" serialized:"destination_cluster,omitempty"`
-	RemoteCluster      string `human:"Remote Cluster" serialized:"remote_cluster,omitempty"`
-	State              string `human:"State" serialized:"state"`
-	Error              string `human:"Error,omitempty" serialized:"error,omitempty"`
-	ErrorMessage       string `human:"Error Message,omitempty" serialized:"error_message,omitempty"`
+	Name               string            `human:"Name" serialized:"link_name"`
+	TopicName          string            `human:"Topic Name" serialized:"topic_name"`
+	SourceCluster      string            `human:"Source Cluster" serialized:"source_cluster,omitempty"`
+	DestinationCluster string            `human:"Destination Cluster" serialized:"destination_cluster,omitempty"`
+	RemoteCluster      string            `human:"Remote Cluster" serialized:"remote_cluster,omitempty"`
+	State              string            `human:"State" serialized:"state"`
+	Error              string            `human:"Error,omitempty" serialized:"error,omitempty"`
+	ErrorMessage       string            `human:"Error Message,omitempty" serialized:"error_message,omitempty"`
+	CategoryCounts     []linkCategoryOut `human:"Mirror Partition States,omitempty" serialized:"category_counts,omitempty"`
+}
+
+type linkCategoryOut struct {
+	StateCategory string `serialized:"state_category"`
+	Count         int32  `serialized:"count"`
```

Comment:
NIt: Maybe add `human` readable description as well?

Reply:
Updated to format which removing the struct to keep it clean for read.